.github/workflows/build-release.yml

```
@@ -0,0 +1,272 @@
+        runs-on: windows-latest
+        if: |
+            github.event_name == 'pull_request' && 
+            startsWith(github.event.pull_request.title, 'chore: release v')
+        timeout-minutes: 30
+        steps:
+            - name: Checkout code
+              uses: actions/checkout@v4
+
+            - name: Extract version from PR title
+              id: version
+              run: |
```

actionlint (1.7.9)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)